src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -6261,6 +6261,9 @@ address generate_avx_ghash_processBlocks() {
       __ cmpl(length, 63);
       __ jcc(Assembler::lessEqual, L_finalBit);
 
+      __ mov64(rax, 0x0000ffffffffffff);
```

Comment:
The constant should have an l suffix.

Reply:
I do not believe this is necessary.  There are multiple occurrences of mov64()s without the `l` suffix.  For example, lines 687-688:
```
    __ mov64(c_rarg3, 0x8000000000000000);
    __ mov64(rax, 0x7fffffffffffffff);
```

Reply:
You are right, the code looks good.